hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/GrpcXceiverService.java

```diff
@@ -57,7 +57,7 @@ public void onNext(ContainerCommandRequestProto request) {
               dispatcher.dispatch(request, null);
           responseObserver.onNext(resp);
         } catch (Throwable e) {
-          LOG.error("{} got exception when processing"
+          LOG.error("Got exception when processing"
                     + " ContainerCommandRequestProto {}: {}", request, e);
           responseObserver.onError(e);
```

Comment:
": {}" should be removed here too as LOG.error take exception as parameter by default.

Reply:
Thank you for the review @xiaoyuyao. Fixed that.